### PR TITLE
demos: check number of channels for net input and image

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/common/samples/ocv_common.hpp [hunk 1]
```diff
@@ -24,6 +24,9 @@ void matU8ToBlob(const cv::Mat& orig_image, InferenceEngine::Blob::Ptr& blob, in
     const size_t width = blobSize[3];
     const size_t height = blobSize[2];
     const size_t channels = blobSize[1];
+    if (static_cast<size_t>(orig_image.channels()) != channels) {
+        THROW_IE_EXCEPTION << "A number of channels for net input and image must match";
```
Comment (IRDonch) [inline demos/common/samples/ocv_common.hpp]: A -> The

Comment (Wovchena) [reply demos/common/samples/ocv_common.hpp]: replaced

**Pull request merged**